src/containerapp/azext_containerapp/_sdk_models.py

```diff
@@ -9263,6 +9263,494 @@ def __init__(
         self.max_replicas = max_replicas
         self.rules = rules
 
+class DaprComponentResiliency(_serialization.Model):
```

Comment:
If your class in 2023-08-01-preview, it should exist already.

Reply:
This is the same as `DaprComponentResiliencyPolicy`

Reply:
Removed it